.gitmodules

```diff
@@ -10,6 +10,16 @@
 [submodule "third_party/tinyobjloader-2.0-rc1"]
 	path = third_party/tinyobjloader-2.0-rc1
 	url = https://github.com/tinyobjloader/tinyobjloader.git
-[submodule "third_party/tinygltf-2.5.0"]
-	path = third_party/tinygltf-2.5.0
-	url = https://github.com/syoyo/tinygltf.git
+[submodule "third_party/cgltf"]
+	path = third_party/cgltf
+	url = https://github.com/jkuhlmann/cgltf.git
+[submodule "third_party/KTX-Software"]
+	path = third_party/KTX-Software
+	url = https://github.com/KhronosGroup/KTX-Software.git
+[submodule "third_party/Vulkan-Headers"]
+	path = third_party/Vulkan-Headers
+	url = https://github.com/KhronosGroup/Vulkan-Headers.git
+[submodule "third_party/stb"]
+	path = third_party/stb
+	url = https://github.com/nothings/stb.git
+	branch = master
```

Comment:
Changed to using cgltf for gltf loading as it supports the required extensions. Since tinygltf is removed then we need a standalone version of stb (as previously it was in tinygltf). Ktx is needed to open and transcode ktx files which also requires the vulkan headers to identify the internal format of each texture (side effect of ktx being Khronos)

Reply:
Pretty happy about that removal of `tinygltf` tbh, it's only "tiny" in the name really...
Did you notice any performance speed-up (or otherwise) with this change?

Reply:
Yes, cgltf is slightly faster than tinygltf in what ive seen.